widget/widgetAssets/locales/en.json

```diff
@@ -0,0 +1,39 @@
+{
+    "checkTranslation": {
+        "HEADER_ADD_CHART":"Add SOC Overview Sankey Chart",
+        "HEADER_EDIT_CHART":"Edit SOC Overview Sankey Chart",
+        "LABEL_TITLE": "Title",
+        "LABEL_DATASOURCE":"Data Source",
+        "LABEL_RECORD_CONTAINING_JSON_DATA":"Record Containing JSON Data",
+        "LABEL_JSON_DATA_SOURCE_MODULES":"JSON Data Source Modules",
+        "LABEL_SELECT_AN_OPTION":"Select an Option",
+        "LABEL_GET_LIVE_DATA":"Get Live Data",
+        "LABEL_SELECT_FIELD":"Select Field",
+        "LABEL_FILTER_RECORD":"Filter Record Which Contains The JSON Data",
+        "LABEL_SELECT_A_MODULE":"Select a module wich contains JSON field",
+        "LABEL_FILTER_CRITERIA":"Filter Criteria",
+        "LABEL_LABELTEXT":"LABEL",
+        "LABEL_SOURCE_NODES":"Source Nodes",
```

Comment:
Here we are selecting a single node value. Please correct string values here to singular.

Same for strings like 'LABEL_SELECT_SOURCE_NODES'